src/components/FontUploader.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import '../styles/FontUploader.css';
+
+const FontUploader: React.FC<{ onFontSelected: (selectedFonts: File[]) => void }> = ({
+  onFontSelected,
+}) => {
+  const [selectedFonts, setSelectedFonts] = useState<File[]>([]);
+
+  const handleFileChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (e.target.files) {
+      const files = Array.from(e.target.files);
+      setSelectedFonts([...selectedFonts, ...files]);
+      onFontSelected([...selectedFonts, ...files]);
+    }
+  };
+
+  const handleDrop = (e: React.DragEvent<HTMLDivElement>) => {
+    e.preventDefault();
+    const files = Array.from(e.dataTransfer.files);
+
+    // Filter out files that do not have the .otf extension
+    const otfFiles = files.filter((file) => file.name.endsWith('.otf'));
```

Comment:
Are all font files .otf? Can't some of them also be .ttf like the ones you download from https://fonts.google.com/ ?